app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -1521,11 +1521,11 @@ private void setAndCalculatePokeSpamText(IVScanResult ivScanResult) {
                     pokeSpamCalculator.getTotalEvolvable(), pokeSpamCalculator.getEvolveRows(),
                     pokeSpamCalculator.getEvolveExtra());
 
-            if (totEvol<3) {
+            if (totEvol < 3) {
```

Comment:
You should make the constant `HOW_MANY_POKEMON_WE_HAVE_PER_ROW` from `Pokespam.java` public and change line 1524 to:
`if (totEvol < Pokespam.HOW_MANY_POKEMON_WE_HAVE_PER_ROW) {`